There is a conflict within the static libcrypto.a with the arc4random,
arc4random_buf, and arc4random_uniform.  These functions also exist in
in FreeBSD's system libc.  To resolve, don't use libressl builtin
versions.  The issue referenced in the patch should be resolved in
current releases.

--- configure.orig	2017-11-05 23:15:42 UTC
+++ configure
@@ -12165,12 +12165,15 @@ fi
 
 
 		;;
+	*dragonfly*)
+		HOST_OS=dragonfly
+		HOST_ABI=elf
+		;;
 	*freebsd*)
 		HOST_OS=freebsd
 		HOST_ABI=elf
 		# fork detection missing, weak seed on failure
 		# https://svnweb.freebsd.org/base/head/lib/libc/gen/arc4random.c?revision=268642&view=markup
-		USE_BUILTIN_ARC4RANDOM=yes
 		PROG_LDADD='-lthr'
 
 		;;
